pom.xml

```diff
@@ -14,141 +14,101 @@
-                            <show>public</show>
-                            <nohelp>true</nohelp>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-
-        <resources>
-            <resource>
-                <filtering>true</filtering>
-                <directory>src/main/resources</directory>
-            </resource>
-        </resources>
-    </build>
-
+                <artifactId>org.jacoco.agent</artifactId>
+                <version>${jacoco.version}</version>
+                <classifier>runtime</classifier>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.easymock</groupId>
+                <artifactId>easymock</artifactId>
+                <version>${easymock.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.powermock</groupId>
+                <artifactId>powermock-api-easymock</artifactId>
+                <version>${powermock.api.easymock.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.powermock</groupId>
+                <artifactId>powermock-module-testng</artifactId>
+                <version>${powermock.module.testng.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon</groupId>
+                <artifactId>org.wso2.carbon.touchpoint.feature</artifactId>
+                <version>${carbon.touchpoint.version}</version>
+                <type>zip</type>
+            </dependency>
+            <dependency>
+                <groupId>org.jacoco</groupId>
+                <artifactId>org.jacoco.ant</artifactId>
+                <version>${org.jacoco.ant.version}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
     <properties>
-        <bundle.activator>org.wso2.carbon.securevault.internal.SecureVaultActivator</bundle.activator>
-        <private.package>org.wso2.carbon.securevault.internal.*,</private.package>
-        <export.package>
-            !org.wso2.carbon.securevault.internal.*,
-            org.wso2.carbon.securevault.*; version="${carbon.kernel.package.export.version}",
-        </export.package>
-        <import.package>
-            org.slf4j.*;version="${slf4j.logging.package.import.version.range}",
-            org.osgi.framework.*;version="${osgi.framework.package.import.version.range}",
-            org.yaml.snakeyaml.*;version="${org.snakeyaml.package.import.version.range}",
-            javax.crypto.*;version="${javax.crypto.version.range}",
-        </import.package>
         <carbon.kernel.package.export.version>5.2.0</carbon.kernel.package.export.version>
```

Comment:
This should be removed since kernel dependeny's on carbon-config and carbon-secvault.